### PR TITLE
use BasicKeep.author instead of BasicKeep.ownerId

### PR DESCRIPTION
**Pull request by camhashemi**
@ryanpbrewster @leogrim

### CODE REVIEW

File: kifi-backend/modules/eliza/app/com/keepit/notify/info/NotificationKindInfoRequests.scala [hunk 1]
```diff
@@ -134,12 +125,12 @@ class NotificationKindInfoRequests @Inject()(implicit val pubIdConfig: PublicIdC
...
         locator = Some(MessageThread.locator(Keep.publicId(event.keepId))),
         extraJson = Some(Json.obj(
-          "keeper" -> keeperOpt,
+          "keeper" -> author,
           "library" -> Json.toJson(libraryKept),
           "keep" -> Json.obj(
             "id" -> newKeep.id,
             "url" -> newKeep.url,
-            "attr" -> Json.toJson(newKeep.attribution)
+            "attr" -> newKeep.attribution
```
Comment (camhashemi) [inline kifi-backend/modules/eliza/app/com/keepit/notify/info/NotificationKindInfoRequests.scala:129]: need to make sure that this doesn't break clients

Comment (camhashemi) [inline kifi-backend/modules/eliza/app/com/keepit/notify/info/NotificationKindInfoRequests.scala:129]: talked with @DerekBurch, looks like this is unused in mobile. looking into the ext.

Comment (camhashemi) [inline kifi-backend/modules/eliza/app/com/keepit/notify/info/NotificationKindInfoRequests.scala:129]: @cvializ @andrewconner do you guys know where `extraJson` is consumed in the ext?

Comment (cvializ) [inline kifi-backend/modules/eliza/app/com/keepit/notify/info/NotificationKindInfoRequests.scala:129]: I can't find any references to it in the source

**Pull request merged**